ndk-build/src/cargo.rs

```diff
@@ -47,7 +47,7 @@ pub fn cargo_ndk(
             }
         }
         Err(std::env::VarError::NotUnicode(_)) => {
-            panic!("CARGO_ENCODED_RUSTFLAGS environment variable contains non-unicode characters")
+            panic!("RUSTFLAGS environment variable contains non-unicode characters")
```

Comment:
Whoops, leave this at `CARGO_ENCODED_RUSTFLAGS`, `RUSTFLAGS` is read above and has its own panic.